golang/pkg/cli/config_file.go

```diff
@@ -280,12 +280,11 @@ func PodmanInfo() {
 }
 
 func DisabledServiceSettings(settings *Settings) *Settings {
-	if settings.Containers.Crux.Disabled {
-		fmt.Printf("Do not forget to add your DATABASE_URL to your crux environment.\n\n")
-		fmt.Printf("DATABASE_URL=postgresql://%s:%s@%s_crux-postgres:%d/%s?schema=public\n\n",
+	if settings.Containers.Crux.Disabled && settings.Command == "up" {
+		log.Info().Msg("Do not forget to add your DATABASE_URL to your crux environment.")
+		log.Info().Msgf("DATABASE_URL=postgresql://%s:%s@localhost:%d/%s?schema=public",
 			settings.SettingsFile.CruxPostgresUser,
 			settings.SettingsFile.CruxPostgresPassword,
-			settings.SettingsFile.Prefix,
```

Comment:
Was this intentional?

Reply:
Deleted `%s_crux-postgres` which contains prefix.

Reply:
https://github.com/dyrector-io/dyrectorio/pull/303/files#diff-150ac8c08a2d6f0433a9edf535a87c5468f513c6aeee343f8c6568edb89f4a7fL363

There are many other things use this

Reply:
I can't open the URL, but what do you think? It's just an info log.

Reply:
`	// Generate names
	settings.Containers.Traefik.Name = fmt.Sprintf("%s_traefik", settings.SettingsFile.Prefix)
	settings.Containers.Crux.Name = fmt.Sprintf("%s_crux", settings.SettingsFile.Prefix)
	settings.Containers.CruxMigrate.Name = fmt.Sprintf("%s_crux-migrate", settings.SettingsFile.Prefix)
	settings.Containers.CruxUI.Name = fmt.Sprintf("%s_crux-ui", settings.SettingsFile.Prefix)
	settings.Containers.Kratos.Name = fmt.Sprintf("%s_kratos", settings.SettingsFile.Prefix)
	settings.Containers.KratosMigrate.Name = fmt.Sprintf("%s_kratos-migrate", settings.SettingsFile.Prefix)
	settings.Containers.CruxPostgres.Name = fmt.Sprintf("%s_crux-postgres", settings.SettingsFile.Prefix)
	settings.Containers.KratosPostgres.Name = fmt.Sprintf("%s_kratos-postgres", settings.SettingsFile.Prefix)
	settings.Containers.MailSlurper.Name = fmt.Sprintf("%s_mailslurper", settings.SettingsFile.Prefix)
`
I meant these. Prefix is used for the name generation for each container. The one you found should have used this, but it wasn't.